### PR TITLE
[alt-ergo-free] Adding extra version for dune >= 3 compatibility

### PR DESCRIPTION
**Pull request by Stevendeo**

### CODE REVIEW

File: packages/alt-ergo-parsers-free/alt-ergo-parsers-free.2.3.3.dune3/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
+license: "Apache-2.0"
+build: [
+  ["./configure" name]
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.04.0" & < "4.07.0"}
+  "dune" {>= "1.5" build <= "3.2.0"}
```
Comment (mseri) [inline packages/alt-ergo-parsers-free/alt-ergo-parsers-free.2.3.3.dune3/opam:12]:
```suggestion
  "dune" {>= "1.5" & <= "3.2.0"}
```

File: packages/alt-ergo-lib-free/alt-ergo-lib-free.2.3.3.dune3/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
+license: "Apache-2.0"
+build: [
+  ["./configure" name]
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.04.0" & < "4.07.0"}
+  "dune" {>= "1.5" build & <= "3.2.0"}
```
Comment (mseri) [inline packages/alt-ergo-lib-free/alt-ergo-lib-free.2.3.3.dune3/opam:12]:
```suggestion
  "dune" {>= "1.5" & <= "3.2.0"}
```

**Comment by Stevendeo** [comment]
I am not sure where the issue is here; the alt-ergo-free.2.3.3 & alt-ergo-free.2.3.3.dune3 are almost identical at this point (except for the suggestions & the additional patch)

File: packages/alt-ergo-parsers-free/alt-ergo-parsers-free.2.3.3.dune3/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
+license: "Apache-2.0"
+build: [
+  ["./configure" name]
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>="4.04.0" & < "4.07.0"}
+  "dune" {>= "1.5" <= "3.2.0"}
```
Comment (kit-ty-kate) [inline packages/alt-ergo-parsers-free/alt-ergo-parsers-free.2.3.3.dune3/opam:12]:
```suggestion
  "dune" {>= "1.5"}
```

**Review by kit-ty-kate — COMMENTED** [review]
these constraints seem rather unnecessary and detrimental to users

**Comment by kit-ty-kate** [comment]
Thanks, sorry for the delay

**Pull request merged**